src/utils/SparseLineGrid.h

```diff
@@ -50,6 +50,7 @@ class SparseLineGrid : public SparseGrid<ElemT>
 
     /*! \brief Accessor for getting locations from elements. */
     Locator m_locator;
+    grid_coord_t sign(grid_coord_t z);
```

Comment:
This would be a nice candidate to go into utils/math.h, but I can also do that.

The general form is:
```
template <typename T> int sgn(T val)
{
    return (T(0) < val) - (val < T(0));
}
```